python: can't open file '/workspaces/01-applied-high-performance-computing/S04/mandelbrot.py': [Errno 2] No such file or directory
python: can't open file '/workspaces/01-applied-high-performance-computing/S04/mandelbrot.py': [Errno 2] No such file or directory
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpiexec detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[47056,1],0]
  Exit code:    2
--------------------------------------------------------------------------
bash: --height: command not found
